app/src/main/java/com/historyquestwaifuedition/models/HotileCreature.kt

```diff
@@ -0,0 +1,10 @@
+package com.historyquestwaifuedition.models
+
+class HotileCreature(
```

Comment:
HotileCreature may need to be changed to HostileCreature

Reply:
Good. Catch. Not sure if we'll need them later on but it's good to think ahead.